ports/effects11/vcpkg.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "effects11",
+  "version-string": "11.26",
+  "port-version": 1,
```

```suggestion
  "port-version": 0,
```
For a new port, this field also should be `0`.